test.sh

```diff
@@ -76,6 +76,11 @@ install_docker_compose(){
 clone_next(){
     rm -rf "$NEXT_DIR"
     git clone https://github.com/inspirehep/inspire-next.git "$NEXT_DIR"
+    # TODO remove once it's merged
+    cd inspire-next
+    git fetch origin refs/pull/2124/head
+    git reset --hard FETCH_HEAD
+    cd -
```

Comment:
It's actually not really needed, as it can go on without the extra vars, so remove.